packages/orbit-components/src/Accordion/AccordionSection/index.js

```diff
@@ -43,7 +43,12 @@ const AccordionSection = ({
           </SectionHeader>
         )}
 
-        <Slide maxHeight={height} expanded={isExpanded} id={slideID} ariaLabelledBy={slideID}>
+        <Slide
+          maxHeight={height}
+          expanded={isExpanded}
+          id={slideID("slideID")}
+          ariaLabelledBy={slideID("slideID")}
```

Comment:
We can decide about the convention in case we need to reuse the same ID, whether we're going to invoke seed inline like this multiple times or save it in variable, like we do in other places. I vote for variable because there's no chance of typos in that case.